### PR TITLE
Updated Vermilion VASP documentation

### PR DESCRIPTION
**Pull request by claralarson**
Better documentation for reliable multi-node jobs with Intel MPI and Open MPI. Need to compare my updates to Tim's updates to make sure there's no overlap.

### CODE REVIEW

File: docs/Documentation/Systems/Vermillion/running.md [hunk 1]
```diff
@@ -335,12 +335,203 @@ task    thread             node name  first task    # on node  core
...
+[myuser@vs example]$ which vasp_std
+/nopt/nrel/apps/220525b/level01/install/opt/spack/linux-rocky8-zen2/gcc-12.1.0/vasp544/bin/vasp_std
+```
+
+Note the directory might be different. 
+
+In order to run on more than one node, we need to specify the network interconnect. To do so, use mpirun instead of srun. We want to use "en7" as the interconnect. The mpirun command looks like this. 
+
+```
+I_MPI_OFI_PROVIDER=tcp mpirun -iface ens7 -np 16 vasp_std
```
Comment (diskwarrior) [inline docs/Documentation/Systems/Vermillion/running.md:380]: iface names don't match. Which one is correct?

Comment (claralarson) [reply docs/Documentation/Systems/Vermillion/running.md:380]: Should be "ens7". Thanks for the catch

Comment (claralarson) [reply docs/Documentation/Systems/Vermillion/running.md]: Fixed it in running.md

**Comment by yandthj** [comment]
Closing, included in PR #210

**Pull request closed without merging**